
float64 timestep
float64 time

int8 predictionCount

geometry_msgs/PoseWithCovariance[] predictions

